Error compiling with GCC 16

```
Compiling rtapi/uspace_rtapi_app.cc
rtapi/uspace_rtapi_app.cc: In static member function ‘static int RtapiApp::allocate_task_id()’:
rtapi/uspace_rtapi_app.cc:118:13: error: ‘ENOSPC’ was not declared in this scope
  118 |     return -ENOSPC;
      |             ^~~~~~
rtapi/uspace_rtapi_app.cc: In member function ‘int RtapiApp::task_new(void (*)(void*), void*, int, int, long unsigned int, int)’:
rtapi/uspace_rtapi_app.cc:133:17: error: ‘EINVAL’ was not declared in this scope
  133 |         return -EINVAL;
      |                 ^~~~~~
rtapi/uspace_rtapi_app.cc: In member function ‘long int RtapiApp::clock_set_period(long int)’:
rtapi/uspace_rtapi_app.cc:191:17: error: ‘EINVAL’ was not declared in this scope
  191 |         return -EINVAL;
```

Adding `#include <errno.h>` to `rtapi/uspace_rtapi_app.cc` fixes the issue.

--- a/src/rtapi/uspace_rtapi_app.cc
+++ b/src/rtapi/uspace_rtapi_app.cc
@@ -21,6 +21,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <errno.h>
 
 std::atomic_int WithRoot::level;
 uid_t WithRoot::ruid, WithRoot::euid;

